### DISCUSSION TITLE
Urgent Help Needed: Account Hacked, Unable to Reach GitHub Support

### DISCUSSION THREAD

**Discussion by asmawi117** [community/community#111053] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Hello everyone,

I hope this message finds you well. Unfortunately, I'm reaching out to the community in desperation as my primary GitHub account has been hacked, and I've been unable to regain control despite numerous attempts to contact GitHub support.

The hacker has changed both the email address and password associated with my account, effectively locking me out of it. I've been trying to reach out to GitHub support for almost a month now, but I haven't received any response.

This situation is incredibly frustrating and concerning for me, as my GitHub account is crucial for my work and personal projects. I've tried all the standard procedures for account recovery, but to no avail.

If anyone has experienced a similar situation or has any advice on how to expedite the resolution process, I would greatly appreciate your assistance. I'm at a loss for what else to do and any help or guidance would be immensely valuable.

Thank you in advance for any assistance you can provide.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @asmawi117, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
